llvm/test/CodeGen/Thumb2/avoidmuls.mir

```diff
@@ -63,5 +63,5 @@ body:             |
 
 ...
```

Comment:
This is probably a better place for this coment:
A lot of the IR in this test case can be removed, to help simplify it a bit. It might be possible to remove the entire ir section and have two run lines that run with/without -mcpu=cortex-m33. That way it can test that both versions with the muls and with the mul are working OK.

Reply:
We also like to use update_mir_test_checks.py where we can, but I'm not sure it's very useful here.

Reply:
I have reduced test body to minimum, and added the non-cortex-m33 case.